cmd/search/http.go

```diff
@@ -19,19 +19,22 @@ import (
 	units "github.com/docker/go-units"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
+
+	"github.com/openshift/ci-search/bugzilla"
 )
 
 type nopFlusher struct{}
 
 func (_ nopFlusher) Flush() {}
 
 type Match struct {
-	Name         string      `json:"name,omitempty"`
-	LastModified metav1.Time `json:"lastModified"`
-	FileType     string      `json:"filename"`
-	Context      []string    `json:"context,omitempty"`
-	MoreLines    int         `json:"moreLines,omitempty"`
-	URL          string      `json:"url,omitempty"`
+	Name         string            `json:"name,omitempty"`
+	LastModified metav1.Time       `json:"lastModified"`
+	FileType     string            `json:"filename"`
+	Context      []string          `json:"context,omitempty"`
+	MoreLines    int               `json:"moreLines,omitempty"`
+	URL          string            `json:"url,omitempty"`
+	BugInfo      *bugzilla.BugInfo `json:"bugInfo,omitempty"`
```

Comment:
Implies that we will keep adding more and more of these.  I guess I can live with it.  Should just be `bug` though, not `bug info`.

Reply:
yeah that was the implication. I figured it might be controversial.

will rename the field.

Reply:
renamed